Zend/tests/accessors/default_on_virtual.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Purely virtual accessors cannot have default value
+--FILE--
+<?php
+
+class Test {
+    private $_prop;
+    public $prop = 0 {
+        get { return $this->_prop; }
+        set { $this->_prop = $value; }
+    }
```

Comment:
```suggestion
    public $prop {
        get { return $this->_prop; }
        set { $this->_prop = $value; }
    } = 0;
```

is allowed by C#, why such syntax is not supported? Supporting default values would be very helpful.